### PR TITLE
Added new open source project

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Fixes Issue

Closes : #735 

## Screenshots

image

#### *Please add required labels of <b>gssoc and assignee</b>

### CODE REVIEW

File: database/opensource/projects.json [hunk 1]
```diff
@@ -54,5 +54,12 @@
     "url": "https://www.gsocorganizations.dev/",
     "category": "open source",
     "subcategory": "open-source-projects"
+  },
+  {
+    "name": "ProjectMate",
+    "description": "It's a open-source platform to connect with experienced maintainers, and collaborate with a community of passionate contributors.",
```
Comment (CBID2) [inline database/opensource/projects.json:60]: [Proofreading]
```suggestion
    "description": "It's an open source platform where new maintainers can connect with experienced maintainers, and collaborate with a community of passionate contributors.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ujjawaltyagii! :) I just made a proofreading suggestion in my inline comment. You’re off to a great start! :)

**Comment by ujjawaltyagii** [comment]
@CBID2 done !!

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by pratyushmp — APPROVED** [review]
Looks good to me!

**Pull request merged**